docs/README.md

```diff
@@ -193,34 +193,33 @@ The version of sphinx theme is fixed in both repositories:
 * https://github.com/apache/airflow-site/blob/main/sphinx_airflow_theme/sphinx_airflow_theme/__init__.py#L21
 * https://github.com/apache/airflow/blob/main/devel-common/pyproject.toml#L77 in "docs" section
 
-In case of bigger changes to the theme, we
-can first iterate on the website and merge a new theme version, and only after that we can switch to the new
+In case of bigger changes to the theme, we can first iterate on the website and merge a new theme version, and only after that can we switch to the new
 version of the theme.
 
 
 # Fixing historical documentation
 
-Sometimes we need to update historical documentation (modify generated `html`) - for example when we find
+Sometimes we need to update historical documentation (modify generated `html`) - for example, when we find
 bad links or when we change some of the structure in the documentation. This can be done via the
 `airflow-site-archive` repository. The workflow is as follows:
 
 1. Get the latest version of the documentation from S3 to `airflow-site-archive` repository using
-   `Sync s3 to GitHub` workflow. This will download the latest version of the documentation from S3 to
-   `airflow-site-archive` repository (this should be normally not needed, if automated synchronization works).
+   `Sync S3 to GitHub` workflow. This will download the latest version of the documentation from S3 to
```

Comment:
Good catch, it's actually named as `Sync S3 to GitHub` here https://github.com/apache/airflow-site-archive/blob/main/.github/workflows/s3-to-github.yml